### ISSUE TITLE
wikibase-cli 17.0.0 build issue

### ISSUE THREAD

**Issue by chenrui333** [maxlath/wikibase-cli#178]
- wikibase-cli version: 17.0.0
- OS version: linux

👋 trying to build the latest release, but run into some build issue. The error log is as below:

<details>
<summary>error build log</summary>

```
==> /home/linuxbrew/.linuxbrew/Cellar/wikibase-cli/17.0.0/bin/wd label Q5 --lang en
  /home/linuxbrew/.linuxbrew/Cellar/wikibase-cli/17.0.0/libexec/lib/node_modules/wikibase-cli/node_modules/commander/index.js:925
          throw new Error(executableMissing);
          ^
  
  Error: 'wd-label' does not exist
   - if 'label' is not meant to be an executable command, remove description parameter from '.command()' and use '.description()' instead
   - if the default executable name is not suitable, use the executableFile option to supply a custom name
      at ChildProcess.<anonymous> (/home/linuxbrew/.linuxbrew/Cellar/wikibase-cli/17.0.0/libexec/lib/node_modules/wikibase-cli/node_modules/commander/index.js:925:15)
      at ChildProcess.emit (node:events:[51](https://github.com/Homebrew/homebrew-core/actions/runs/5581607149/jobs/10199977796?pr=136851#step:4:52)2:28)
      at ChildProcess._handle.onexit (node:internal/child_process:291:12)
      at onErrorNT (node:internal/child_process:483:16)
      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

</details>

full build log, https://github.com/Homebrew/homebrew-core/actions/runs/5581607149/jobs/10199977796?pr=136851
relates to Homebrew/homebrew-core#136851

**Comment by maxlath** [comment]
Thanks for reporting, this should now have been fixed with fa07c6b, published in `v17.0.1`

**Comment by chenrui333** [comment]
17.0.1 works for me, thanks @maxlath!
